tree/treeplayer/inc/ROOT/TDFActionHelpers.hxx

```diff
@@ -92,6 +93,30 @@ public:
    ULong64_t &PartialUpdate(unsigned int slot);
 };
 
+template<typename ProxiedVal_t>
+class ReportHelper {
+   const std::shared_ptr<TCutFlowReport> fReport;
+   std::weak_ptr<ProxiedVal_t> fProxiedWPtr;
+   bool fReturnEmptyReport;
```

Comment:
we should probably comment why we need this so we don't forget ourselves :)

Reply:
:-) We have so much time and so little to do that I tend to forget these things. Thanks for the reminder!!